generic-ec/Cargo.toml

```diff
@@ -14,7 +14,7 @@ keywords = ["elliptic-curves"]
 
 [dependencies]
 generic-ec-core = "0.1"
-generic-ec-curves = { version = "0.1", optional = true }
+generic-ec-curves = { path = "../generic-ec-curves", optional = true }
```

Comment:
Note: this is what I was talking about when I said it's stupid in cargo that selecting a package and selecting where to get the package are not be separate

Reply:
It looks fine to me :thinking: